pkg/new-ui/v1beta1/frontend/src/app/constants/algorithms-types.const.ts

```diff
@@ -10,6 +10,7 @@ export const AlgorithmNames = {
   [AlgorithmsEnum.BAYESIAN_OPTIMIZATION]: 'Bayesian Optimization',
   [AlgorithmsEnum.TPE]: 'Tree of Parzen Estimators',
   [AlgorithmsEnum.CMAES]: 'Covariance Matrix Adaptation: Evolution Strategy',
+  [AlgorithmsEnum.SOBOL]: 'Sobol Quasirandom Sequence',
```

Comment:
I cheked the new Katib UI.

![ScreenShot 2021-04-23 21 52 44](https://user-images.githubusercontent.com/5564044/115889932-619eae80-a48f-11eb-9cce-cb23bad76590.png)